api/rest/iot.connection.def.js

```diff
@@ -32,7 +32,7 @@ module.exports = function(config) {
      * Top of the hierarchy. Common attributes to every
      * Connection Options
      */
-    function ConnectionOptions() {
+    function ConnectionOptions(isBinary) {
```

Comment:
I don't think the "isBinary" flag needs to be sent to this function as you are overriding the body and headers whenever this is used anyway so what gets set here is irrelevant...